src/main/java/com/salesforce/storm/spout/dynamic/metrics/StormRecorder.java

```diff
@@ -176,6 +176,11 @@ public void assignValue(final Class sourceClass, final String metricName, final
     public void timer(final Class sourceClass, final String metricName, final long timeInMs) {
         final String key = generateKey(sourceClass, metricName);
         timers.scope(key).update(timeInMs);
+
+        // Update total time value.  This tracks how much time in total has been
+        // spent in this key, in milliseconds
+        final String totalTimeKey = metricName + "_totalTimeMs";
```

Comment:
This is kind of cool, now we'll record both the average time for the metric, but also the total time spent in that area during a run under a counter.

So you'll have both:
my metric ran on average within 1242.2 milliseconds, and in total spend 2342342349ms